lib/helper/get.helper.js

```diff
@@ -0,0 +1,11 @@
+const get = (obj, path, defaultValue = undefined) => {
```

Comment:
Best verwijzen naar docs:

```suggestion
// See https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_get
const get = (obj, path, defaultValue = undefined) => {
```

Reply:
comment toegevoegd